### DISCUSSION TITLE
Where can I see all possible configuration options fo the config.yaml

### DISCUSSION THREAD

**Discussion by jie-Guo** [rancher/rke2#4293] [Q&A] [answered]
this url is not found： https://docs.rke2.io/install/install_options/install_options/#configuration-file

like this url : https://docs.rke2.io/reference/server_config
i want to see a full options of /etc/rancher/rke2/config.yaml，where can i find it?

**Comment by brandond** [comment, marked as answer]
Where did you find that link that does not work?

All valid CLI flags are valid config file entries. You can find CLI help at:
* https://docs.rke2.io/reference/linux_agent_config
* https://docs.rke2.io/reference/server_config

This is described at https://docs.rke2.io/install/configuration
> The configuration file parameters map directly to CLI arguments, with repeatable CLI arguments being represented as YAML lists. Boolean flags are represented as true or false in the YAML file.
